### ISSUE TITLE
MATE Mouse pointer style unavailable

### ISSUE THREAD

**Issue by sixwheeledbeast** [lah7/Ambiant-MATE#12]
### Describe the problem
On fresh installs of Ubuntu-MATE, after installation of the "proper" MATE themes via your PPA (Thank you)
MATE version of the mouse pointer is unavailable only the awful Yaru one...

### Expected behaviour
"MATE" cursor option available in Appearance > Customise > Pointer
Files formally found in /usr/share/icons/mate/cursors
Selecting a MATE theme should also automatically have the MATE cursor style.

### Affected Themes

- Ambiant-MATE
- Ambiant-MATE-Dark
- Radiant-MATE

### Operating System / Environment
Ubuntu-MATE 22.04 LTS

Apologies, if I have posted this issue in the wrong place.
I assume it would be an oversight that this isn't pulled in as part of the theme?

**Comment by lah7** [comment]
Don't worry, you've reached the right place 👍🏼 

`index.theme` file is correctly specifying to use the `mate` icon theme:

https://github.com/lah7/Ambiant-MATE/blob/df003b1d815626b7ba4dc78c5fafce9a370148c0/usr/share/themes/Ambiant-MATE/index.theme#L11-L12

However, it turns out that these cursors are part of the `mate-icon-theme` package, which is no longer pre-installed in Ubuntu MATE. I'll add a dependency on that package, which will fix the problem.

**Comment by sixwheeledbeast** [comment]
Ah great thanks. That makes sense. 82MB is not too much of an issue but I can understand wanting to keep things light.
Would it create an issue or install conflicts having them bundled into the icon theme then installing mate-themes or vice-versa?

**Comment by lah7** [comment]
Speedy! I did edit my comment, It looks like it would be simpler to depend on the package. Didn't realise **one** of the `cursors` folders is about 32.7 MB on its own.

Then to top it off, `ambiant-mate-gtk-themes` should depend on `ambiant-mate-icon-themes` (of which depends on `mate-icon-themes`) so the complete theme will work as expected out of the box.

**Comment by lah7** [comment]
> Would it create an issue or install conflicts having them bundled into the icon theme then installing mate-themes or vice-versa?

If we did go that route, the cursors would be added to our `Ambiant-MATE` (and `-Dark`) icon themes, so no conflicts, but it would probably waste disk space having duplicates installed.

That could be a plan in future if the MATE icon theme package has a radical change (like redesigned cursors)

**Comment by lah7** [comment]
Done! A new package (`22.04.2-2`) has been dispatched. It'll take some time for Launchpad to publish it. Thanks for spotting!

**Comment by sixwheeledbeast** [comment]
So I have just upgraded the package but MATE pointers are still unavailable even after a reboot.
dpkg shows **ambiant-mate-gtk-themes** **ambiant-mate-icon-themes** **mate-icon-themes** as all installed.
_/usr/share/icons/mate/cursors_ doesn't exist if that's helpful.

**Comment by lah7** [comment]
You're right, I made a mistake in the dependencies! It's actually  `mate-themes` that provides `/usr/share/icons/mate/cursors`, which is confusingly stored in the same folder as `mate-icon-themes`, so I ended up putting the wrong dependency.

Sorry about that, and thanks again for checking! The `22.04.2-3` package will fix for real this time. Checked under a `ubuntu-mate-22.04.3-desktop-amd64.iso` live session.
